scripts/classification/imagenet/train_imagenet.py

```diff
@@ -9,102 +16,243 @@
-                        help='size of the input image size. default is 224')
-    parser.add_argument('--crop-ratio', type=float, default=0.875,
-                        help='Crop ratio during validation. default is 0.875')
-    parser.add_argument('--use-pretrained', action='store_true',
-                        help='enable using pretrained model from gluon.')
-    parser.add_argument('--use_se', action='store_true',
-                        help='use SE layers or not in resnext. default is false.')
-    parser.add_argument('--mixup', action='store_true',
-                        help='whether train the model with mix-up. default is false.')
-    parser.add_argument('--mixup-alpha', type=float, default=0.2,
-                        help='beta distribution parameter for mixup sampling, default is 0.2.')
-    parser.add_argument('--mixup-off-epoch', type=int, default=0,
-                        help='how many last epochs to train without mixup, default is 0.')
-    parser.add_argument('--label-smoothing', action='store_true',
-                        help='use label smoothing or not in training. default is false.')
-    parser.add_argument('--no-wd', action='store_true',
-                        help='whether to remove weight decay on bias, and beta/gamma for batchnorm layers.')
-    parser.add_argument('--teacher', type=str, default=None,
-                        help='teacher model for distillation training')
-    parser.add_argument('--temperature', type=float, default=20,
-                        help='temperature parameter for distillation teacher model')
-    parser.add_argument('--hard-weight', type=float, default=0.5,
-                        help='weight for the loss of one-hot label for distillation training')
-    parser.add_argument('--batch-norm', action='store_true',
-                        help='enable batch normalization or not in vgg. default is false.')
-    parser.add_argument('--save-frequency', type=int, default=10,
-                        help='frequency of model saving.')
-    parser.add_argument('--save-dir', type=str, default='params',
-                        help='directory of saved models')
-    parser.add_argument('--resume-epoch', type=int, default=0,
-                        help='epoch to resume training from.')
-    parser.add_argument('--resume-params', type=str, default='',
-                        help='path of parameters to load from.')
-    parser.add_argument('--resume-states', type=str, default='',
-                        help='path of trainer state to load from.')
-    parser.add_argument('--log-interval', type=int, default=50,
-                        help='Number of batches to wait before logging.')
-    parser.add_argument('--logging-file', type=str, default='train_imagenet.log',
-                        help='name of training log file')
-    parser.add_argument('--use-gn', action='store_true',
-                        help='whether to use group norm.')
+    parser = argparse.ArgumentParser(
```

Comment:
I don't think this code style change is necessary. @hetong007

Reply:
@zhanghang1989  sorry for that, but every time im saving python file it runs code formatter (black) which im using in my daily work. Of course it makes no sense to have only one/two file formatted. Anyway I had to edit `train_imagenet` to import back GroupNorm from opencv, not mxnet as i had in previous PR.

Reply:
@mjamroz is there a way to pause your code formatter? The style changes are indeed unexpected, and they make review a bit more challenging (too many diffs).

Reply:
ok, next try @hetong007  @zhanghang1989 
https://github.com/dmlc/gluon-cv/pull/1216